sdk/eventhub/azure-eventhubs/examples/iothub_recv.py

```diff
@@ -9,20 +9,17 @@
 An example to show receiving events from an IoT Hub partition.
 """
 import os
-import logging
 
 from azure.eventhub import EventHubClient, EventPosition
 
-
-logger = logging.getLogger('azure.eventhub')
-
 iot_connection_str = os.environ['IOTHUB_CONNECTION_STR']
 
 client = EventHubClient.from_connection_string(iot_connection_str, network_tracing=False)
-consumer = client.create_consumer(consumer_group="$default", partition_id="0", event_position=EventPosition("-1"), operation='/messages/events')
+consumer = client.create_consumer(consumer_group="$default", partition_id="0", event_position=EventPosition("-1"),
+                                  operation='/messages/events')
 with consumer:
     received = consumer.receive(timeout=5)
     print(received)
 
-    eh_info = client.get_properties()
-    print(eh_info)
+eh_info = client.get_properties()
+print(eh_info)
```

Comment:
is this print needed?

Reply:
I removed these two line of codes related to get_properties